packages/MarkdownEditor-Tests.package/MarkdownBlockTextStylerTest.class/instance/assert.parsesToRuns.values..st

```diff
@@ -0,0 +1,10 @@
+accessing
+assert: markdownString parsesToRuns: runs values: values
+
+	| text |
+	text := markdownString asText.
```

Comment:
This is still the incorrect parameter order for the assert statement. What about `assertRuns: runs values: values areStyledResultOf:`? This should imply that the string will be styled...

Reply:
Correct. We have in the heading tests also one confused order. (#10) But I would open a second PR for it, because this one gets to large to put more stuff into it...